docs/getting-started/provision-infrastructure.md

```diff
@@ -0,0 +1,224 @@
+You can then delete the `RDSInstance`:
+```
+kubectl delete -f aws-rds.yaml
+```
+
+## GCP
+
+The GCP provider supports provisioning a [CloudSQL] instance with the
+`CloudSQLInstance` CRD it installs into your cluster. Create a new file named
+`cloudsql.yaml` with the following contents:
+```yaml
+apiVersion: database.gcp.crossplane.io/v1beta1
+kind: CloudSQLInstance
+metadata:
+  name: cloudsqlmysql
+specTemplate:
+  forProvider:
+    databaseVersion: MYSQL_5_7
+    region: us-central1
+    settings:
+      tier: db-n1-standard-1
+      dataDiskType: PD_SSD
+      dataDiskSizeGb: 10
+  writeConnectionSecretsToRef:
+    namespace: crossplane-system
+    name: cloudsqlmysql-conn
+  providerRef:
+    name: gcp-provider
+  reclaimPolicy: Delete
+```
+
+Create the `CloudSQLInstance`:
+```
+kubectl apply -f cloudsql.yaml
+```
+
+You can view the progress of provisioning the database with the following
+command:
+```
+kubectl get -f cloudsql.yaml
+```
+
+When provisioning is complete, you should see `STATUS: Unbound` in the output.
+You can then delete the `CloudSQLInstance`:
+```
+kubectl delete -f cloudsql.yaml
+```
+
+## Azure
+
+The Azure provider supports provisioning an [Azure Database for MySQL] instance
+with the `MySQLServer` CRD it installs into your cluster. Create a new file
+named `mysqlserver.yaml` with the following contents:
+```yaml
+apiVersion: database.azure.crossplane.io/v1beta1
+kind: MySQLServer
+metadata:
+  name: sqlservermysql
+spec:
+  forProvider:
+    administratorLogin: myadmin
+    resourceGroupName: <your-resource-group> # replace with resource group of your choosing
+    location: West US 2
+    sslEnforcement: Disabled
+    version: "5.7"
+    sku:
+      tier: GeneralPurpose
+      capacity: 2
+      family: Gen5
+    storageProfile:
+      storageMB: 20480
+  writeConnectionSecretsToRef:
+    namespace: crossplane-system
+    name: sqlservermysql-conn
+  providerRef:
+    name: azure-provider
+  reclaimPolicy: Delete
+```
+
+Create the `MySQLServer`:
+```
+kubectl apply -f mysqlserver.yaml
+```
+
+You can view the progress of provisioning the database with the following
+command:
+```
+kubectl get -f mysqlserver.yaml
+```
+
+When provisioning is complete, you should see `STATUS: Unbound` in the output.
+You can then delete the `MySQLServer`:
+```
+kubectl delete -f mysqlserver.yaml
+```
+
+## Alibaba
+
+
+The Alibaba provider supports provisioning an [AsparaDB for RDS] instance with
+the `RDSInstance` CRD it installs into your cluster. Create a new file named
+`alibaba-rds.yaml` with the following contents:
+```yaml
+apiVersion: database.alibaba.crossplane.io/v1alpha1
+kind: RDSInstance
+metadata:
+  name: rdsmysql
+spec:
+  forProvider:
+    engine: mysql
+    engineVersion: "10.0"
+    dbInstanceClass: rds.mysql.s1.small
+    dbInstanceStorageInGB: 20
+    securityIPList: "0.0.0.0/0"
+    masterUsername: "test123"
+  writeConnectionSecretsToRef:
```

Comment:
where is this defined? I can't seem to see in the [Alibaba provider](https://github.com/crossplane/provider-alibaba/tree/master/apis) types.

Reply:
@ryanzhang-oss this was adapted from https://github.com/crossplane/provider-alibaba/blob/master/cluster/examples/database/resource_class.yaml

Reply:
@ryanzhang-oss It comes from the embedded `ResourceSpec`, which is common to all Crossplane managed resources.

* https://github.com/crossplane/provider-alibaba/blob/b4b04d/apis/database/v1alpha1/rds_types.go#L94
* https://github.com/crossplane/crossplane-runtime/blob/290de/apis/core/v1alpha1/resource.go#L175